fdmi/plugins/fdmi_app

```diff
@@ -375,5 +497,7 @@ if __name__ == '__main__':
                     type=str)
     ap.add_argument('-sf', '--process-fid', required=False, help='Process FID',
                     type=str)
+    ap.add_argument('-ks', '--kafka-server', required=False, help='kafka server and port',
```

Comment:
Do we change the python code to be a kafka producer?

Reply:
Yes.

This python script is a kafka producer, and sending FDMI events (as messages) to kafka server. So, the Replication manager will get these FDMI events.

This python script also starts an internal kafka consumer in a thread. This is purely for ACK.
Please see my comments above.

Reply:
What I am thinking is as follows:

 Once an FDMI event is received:
		- if it is a duplicate, ignore, send FDMI ack
		- if not a duplicate, update duplicate cache, and call kafka producer
	In the background, producer callback func:
			- send FDMI ack on completion

Reply:
OK. Understood.
I will fix it.